test/api.js

```diff
@@ -2,7 +2,10 @@ const test = require('ava');
 const sinon = require('sinon');
 const { APIClient, SendEmailRequest } = require('../api');
 
-const apiRoot = 'https://api.customer.io/v1';
+const apiRoot = {
```

Comment:
I don't think we gain any extra safety by repeating the object here. Should we just import `apiRoot` from the common file? (same for the other test file)

Reply:
True, I'll make the change to import.